### PR TITLE
initial history docs commit

### PR DESCRIPTION
**Pull request by roneli**

### CODE REVIEW

File: docs/history.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+- **Retention** [Optional]: defines the retention policy of your data and how long it should exist in days, by default is 7 day. 
+- **TimeInterval** [Optional]: Defines how history chunks are split by time, defaults to one chunk per 24 hours. See TimescaleDB [docs](https://docs.timescale.com/api/latest/distributed-hypertables/create_distributed_hypertable/#sample-usage) for more info on this behavior.
+- **TimeTruncation** [Optional]: truncates fetch time by hour, for example if we fetch with TimeTruncation = 1 at 11:25 the fetch date will truncate to 11:00, defaults to 24 hours, which means one set of fetch data per day. Data fetched on the **same** truncation date will be replaced.
+
+
+```
+cloudquery 
+  # history block configuration makes cloudquery run in history mode
+  history {
```
Comment (yevgenypats) [inline docs/history.md:36]: When we do `cloudquery init` do we have defaults? I suggest to output those defaults as well and if it's not a timescaledb just to ignore them and print an info/warn

File: docs/history.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+CloudQuery will create a new schema called `history` which holds full historical data for each table. In our public schema we have `views` that fetch the **latest** fetch data of each tabls. This allows your previous queries before enabling history mode to work out of the box. 
+
+```SQL
+
+-- Will select all historical fetches of aws_iam_users
+SELECT * FROM history.aws_iam_users
+
+-- Will select the latest fetch of aws_iam_users
+SELECT * from aws_iam_users
```
Comment (yevgenypats) [inline docs/history.md:56]: I'll add another example with date

File: sidebars.js [hunk 3]
```diff
@@ -113,6 +113,11 @@ module.exports = {
         'developers/migrations',
       ],
     },
+    {
+      type: 'doc',
```
Comment (yevgenypats) [inline sidebars.js:117]: I think at the current structure it should be under `cli/fetch`

**Review by yevgenypats — CHANGES_REQUESTED** [review]
few comments

File: docs/history.md (hunk 1 above)
Comment (roneli) [reply docs/history.md:36]: only if this block is defined, history is enabled.

**Pull request closed without merging**